src/renderer/modules/webpack/get-modules.ts

```diff
@@ -65,6 +65,62 @@ export function getExportsForProps<T, P extends PropertyKey = keyof T>(
   }
 }
 
+/**
```

Comment:
How is this different from the one we already have above? Would much rather modify that to be reusable rather than create another one which is basically the same.

Reply:
if you mean iterateModuleFunctions function it gets functions from the first level and second level of the object. I didn't knew how to keep this inside an already existing function while keeping it have same functionality so did this

Reply:
Add a parameter to control which behavior it should use

Reply:
done